static/js/common.js

```diff
@@ -15,10 +15,17 @@ const emailInput = document.getElementsByClassName('successEmail')[0];
 
 const nearYouButton = document.getElementsByClassName('nearYouButton')[0];
 
+const mobileThreshold = 768;
+
 var imageFile;
 var selectedLogo;
 var selectedLogoId;
 var emailCode;
 
 var pointersOnMap = [];
 var logoIcons = [];
+
+// Define the function before the event listener
+function isMobile() {
```

Comment:
There are more official methods to determine if you are on mobile. However, imagine a flip phone held horizontally: we would like to treat it as a desktop I guess. So this is fine

Reply:
There are indeed methods to [detect a mobile browser](https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser), but that doesn't really provide anything practical. As you said, a phone held horizontally would be better to treat as desktop. So therefore I will keep my current implementation.